src/crypto/CHIPCryptoPALmbedTLS.cpp

```diff
@@ -941,7 +941,6 @@ CHIP_ERROR P256Keypair::Deserialize(P256SerializedKeypair & input)
 	keypair			= nullptr;
 	mInitialized	= true;
 
-exit:
```

Comment:
why is the "exit" target removed? It looks like it also existed before tinycrypt, or maybe I am looking in the wrong location?

Reply:
Check the patched files, it is not used and if you compile with is_debug=true, it will fail with error of label not used.

Reply:
Thanks, the "label not used" error appears when compiling with tinycrypt enabled and is_debug=true.